src/pages/ExamsPage/hooks.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import { RequestKeys } from 'data/constants';
+
+import * as reduxHooks from 'data/redux/hooks';
+
+import * as api from './data/api';
+import * as selectors from './data/selectors';
+import * as reducer from './data/reducer';
+
+import * as module from './hooks'; // eslint-disable-line import/no-self-import
+
+export const state = {
+  // example of component state that does not need to be in redux
+  exampleValue: (val) => React.useState(val), // eslint-disable-line
```

Comment:
values that won't be needed across multiple components don't need to be in redux, this scaffolds out an approach for maintaining local component state in a hooks file.

Reply:
just a placeholder so we have a pattern to copy when we implement the actual UI stories